chain/indexer/src/block.rs

```diff
@@ -204,8 +203,7 @@ where
                 }
 
                 if tx.is_some() {
-                    let indexing_scope = chain_head - latest_block_in_db;
-                    let progress = 1_f64 - ((chain_head - current_block) as f64 / (indexing_scope as f64));
+                    let progress = (current_block - genesis) as f64 / (chain_head - genesis) as f64;
```

Comment:
An even better version would be: `(current_block - latest_block_in_db) as f64 / (chain_head - latest_block_in_db) as f64`